addons/account/models/account_move.py

```diff
@@ -4145,36 +4146,19 @@ def write(self, vals):
 
         if not self.env.context.get('tracking_disable', False):
             # Create the dict for the message post
-            tracking_values = {}  # Tracking values to write in the message post
+            tracking_value_ids_list = {}
```

Comment:
This names suggests that `tracking_value_ids_list` is a list of tracking value ids, which isn't the case.